assets/js/plugin-check-admin.js

```diff
@@ -13,6 +13,21 @@
 		return;
 	}
 
+	// Handle disabling the check it button when a plugin is not selected.
+	pluginsList.addEventListener( 'change', canRunChecks );
```

Comment:
Nit-pick: Maybe move this together with the `canRunChecks()` call, just to have those two places where the function is called co-located?

Reply:
Thanks @felixarntz I've moved this down with the `canRunChecks()` call.